connectors/src/connectors/confluence/index.ts

```diff
@@ -227,10 +226,9 @@ export async function setConfluenceConnectorPermissions(
     }
   }
 
-  // TODO(2024-01-10 flav) Uncomment in next PR.
-  // if (shouldFullSync) {
-  //   await launchConfluenceFullSyncWorkflow(connectorId, null);
-  // }
+  if (shouldFullSync) {
+    await launchConfluenceFullSyncWorkflow(connectorId, null);
```

Comment:
We need to check the return code of that function to return an error if the workflow has not been started.

We usually use a Ok / Err at this level.

Reply:
Fixed.